### ISSUE TITLE
Install dedicated version of a package

### ISSUE THREAD

**Issue by natefabian18** [robertZaufall/ChocolateStoreCore#57]
Would it be possible to specify a specific line in the download list to be of a specific version. Im trying to install a nodejs application offline and would need the LTS version of nodejs rather than the experimental. Similar to how you can install a dedicated version though the `choco` command with `choco install nodejs --version 20.15.0`

**Comment by robertZaufall** [comment]
It's sure possible but I did not have the usecase myself by now.  

Nodejs is an easy case to do it by hand by the way.  
You can download the `nodejs.install`-nupkg directly from the chocolatey repository, as the binaries are embedded in the file and the package has not to be modified for local storage.  
`nodejs.install` is sufficient - you don't need the `nodejs` package.

**Comment by gggggeorge** [comment]
First, let me say **ChocolateStoreCore is very cool** (I was doing basically the same thing, but editing the nupkg packages myself -which is not so much fun).

----
**I'm all for this as well (ability to specify a version).**

Here is a real-world example, I just encountered:

I just tried downloading **firefox**, and ChocolateStoreCore.exe successfully gets version **127.0.2**:

![image](https://github.com/robertZaufall/ChocolateStoreCore/assets/43475376/1245cd25-05a7-4050-868d-33cf80b057da)

----
I go to [https://community.chocolatey.org/packages/Firefox](https://community.chocolatey.org/packages/Firefox)

-and the latest is actually **128.0.0**, as per (released yesterday (07/09/2024):

![image](https://github.com/robertZaufall/ChocolateStoreCore/assets/43475376/6608cf3c-fdd9-4363-8dfb-c79228c29224)

----
So, to avoid this sort of ambiguity of whatever the logic is (caching, 2nd to the latest version, etc.), could we simply have the ability to indicate the (optional) version in `download.txt`:

`firefox 128.0.0`

Some reasons why:

1. maybe I only want version x.y.z, a few versions older than the latest/greatest.

2. if I mess up (I accidentally delete my "store" dir), I can just run ChocolateStoreCore.exe, and it will do exactly what it did say, three weeks ago, when I ran it -this is great for repeatability.

----
For context, I'm using ChocolateStoreCore.exe to create the "corrected" packages (meaning external binaries are now local downloads), in the context of a sonatype nexus service:

- hosted: where I upload the packages created by ChocolateStoreCore.exe.
- proxy: where I get packages without external binary references -from chocolatey.org
- group: binds hosted & proxy together -and the group is where my local chocolatey install(s) gets their packages from, and the "group" is behind an internet firewall.


Thanks,
George H.,
07/10/2024 4:07pm ET
